See: https://github.com/proot-me/PRoot/issues/106

--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -105,7 +105,7 @@
 		then /bin/echo -e "\#undef VERSION\n\#define VERSION \"$${VERSION}\""; \
 		fi;
 
-CHECK_FEATURES = process_vm seccomp_filter
+CHECK_FEATURES = process_vm
 CHECK_PROGRAMS = $(foreach feature,$(CHECK_FEATURES),.check_$(feature))
 CHECK_OBJECTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).o)
 CHECK_RESULTS  = $(foreach feature,$(CHECK_FEATURES),.check_$(feature).res)
